modules/Sources/Features/Home/HomeView.swift

```diff
@@ -112,8 +116,19 @@ extension HomeView {
         .padding(.bottom, 30)
         .disable(
             when: viewStore.isSendButtonDisabled,
-            dimmingOpacity: 0.5
+            dimmingOpacity: 0.8
         )
+    
+    }
+    
+    func dismissButton(_ viewStore: HomeViewStore) -> some View {
+        Button(action: {
+            viewStore.send(.updateDestination(.send))
+        }, label: {
+            Text(L10n.Home.sendZec(tokenName))
+        })
```

Comment:
You can omit action: and label: here

Reply:
Name of the button is incorrect, it's not a `dismissButton`. According to the modifier `disableButtonStyle` applied it's a disabledButton.